fabfile/use.py

```diff
@@ -19,8 +19,12 @@ def citus(*args):
 
     if len(args) != 1:
         abort('You must provide a single argument, with a command such as "use.citus:v6.0.1"')
+    
     git_ref = args[0]
 
+    # register that we use community
+    config.settings['community-enterprise'] = 'community'
```

Comment:
Instead of using hard coded strings in different places, can you create variables for these? That way, it is much less error prone. You can put those variables in config. So the motivation is that we should have one place to change them. Currently if you want to change this `community` string you would need to change it at different places.

Also what does `git_ref` hold? Doesnt it already have something like `citus` or `enterprise` ?

Reply:
> Instead of using hard coded strings in different places, can you create variables for these? That way, it is much less error prone. You can put those variables in config. So the motivation is that we should have one place to change them. Currently if you want to change this community string you would need to change it at different places.

done

> Also what does git_ref hold? Doesnt it already have something like citus or enterprise ?

It only keeps name of branch and for some cases, we may not be able to identify branches' repository (like release branches)